### ISSUE TITLE
support get_post_meta() last argument: $single

### ISSUE THREAD

**Issue by davidmosterd** [codepress/admin-columns-issues#226]
There are 3 methods to store post meta:
- single value per key
- serialised
- multiple values per key

The last option is very valid and it would be nice when you select a custom field for a column you can tell AC that there might be multiple entries in the database for this key.

No doubt this makes sorting and what kind of data to expect a more difficult task, but the end-user might still think this a value feature. Normally a taxonomy would solve this issue, but there are situations where post meta is just more flexible. 

So in short: It would allow a post where under they key "colors" the values "red", "blue" and "green" are stored to display them all in the column.
